src/udiskslinuxfilesystem.c

```diff
@@ -183,6 +186,13 @@ udisks_linux_filesystem_update (UDisksLinuxFilesystem  *filesystem,
   dev = udisks_linux_block_object_get_device_file (object);
   type = g_udev_device_get_property (device->udev_device, "ID_FS_TYPE");
   size = 0;
+
+  //skip get_info when drive is suspended
+  if (!get_pm_state(device, &error, &count))
+      goto out;
+  if (!(count == 0x80 || count == 0xff))
```

Comment:
If these are standardized values, please either add a comment or make them constants so that it's clear, what they represent.